CHANGELOG.md

```diff
@@ -5,6 +5,21 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org).
 
+## [v3.20.0](https://github.com/puppetlabs/puppetlabs-peadm/tree/v3.20.0) - 2024-07-01
+
+[Full Changelog](https://github.com/puppetlabs/puppetlabs-peadm/compare/v3.19.0...v3.20.0)
+
+### Added
+
+- Add backup/restore plans [#339](https://github.com/puppetlabs/puppetlabs-peadm/pull/339) ([timidri](https://github.com/timidri))
+
+### Other
+
+- [ITHELP-87329] Update test-backup-restore.yaml [#447](https://github.com/puppetlabs/puppetlabs-peadm/pull/447) ([binford2k](https://github.com/binford2k))
+- [ITHELP-87329] Update test-backup-restore.yaml [#446](https://github.com/puppetlabs/puppetlabs-peadm/pull/446) ([binford2k](https://github.com/binford2k))
+- (PE-37233) Adding add_compiler to test matrix [#434](https://github.com/puppetlabs/puppetlabs-peadm/pull/434) ([ragingra](https://github.com/ragingra))
```

Comment:
I'm not a big fan of jira keys that reference internal tickets. It's always annoying when those contain more information but to the majority of people the information isn't accessible.

Reply:
Usually the commit title and description should cover the general purpose and what has changed. Unfortunately not perfect not having public visibility of the Jira tickets, but just for internal tracking.